simple-node-promise/server.js

```diff
@@ -0,0 +1,16 @@
+var http = require("http");
+
+var { requestHandlers } = require("./handlers");
+
+http.createServer((req, res) => {
+    const url = req.url;
+
+    if (url === "/") {
+        requestHandlers.welcome(res)
+    } else if (url === "/data") {
+        requestHandlers.getData(res)
+    } else {
+        requestHandlers.handle404(res)
+    }
+
```

Comment:
Should have a log here to notify server was started to succeed. Notify info should have port